nautobot/docs/plugins/development.md

```diff
@@ -609,6 +610,89 @@ secrets_providers = [ConstantValueSecretsProvider]
 
 After installing and enabling your plugin, you should now be able to navigate to `Secrets > Secrets` and create a new Secret, at which point `"constant-value"` should now be available as a new secrets provider to use.
 
+### Extending Filters
+
+Plugins can extend the current FilterSet's and FilterForm's that are natively provided.
+
+The basic requirements (additional requirements for each are described below) to extend a filter set or a filter form are:
```

Comment:
```suggestion
The basic requirements to extend a filter set or a filter form are:
```

Reply:
How does this sound?: 
The basic requirements that extend to both the extension filter set and a filter form are: